perf/benchmark/runner/graph.py

```diff
@@ -69,8 +69,11 @@ def generate_chart(mesh, csv, x_label, y_label_short, charts_output):
 
     # 5. create chart -> save as interactive HTML
     p = build_chart(title, x_label, x_series, y_label, y_label_short, y_series)
-    show(p)
-    print("HTML graph saved at %s" % f)
+    if show_graph:
+        show(p)
+    else:
+        save(p)
+    print("HTML graph saved at %s" % charts_output)
```

Comment:
this should be under the `else` clause ?

Reply:
done